src/Storefront/Theme/ThemeCompiler.php

```diff
@@ -214,8 +216,20 @@ private function compileStyles(
             );
         }
         $autoPreFixer = new Autoprefixer($cssOutput);
-
-        return $autoPreFixer->compile();
+        $cssOutput = $autoPreFixer->compile();
```

Comment:
I would prefer an own class which extends from https://github.com/padaliyajay/php-autoprefixer/blob/master/src/Autoprefixer.php#L16` and add this fix https://github.com/padaliyajay/php-autoprefixer/commit/475d41b7ee4213de6ce869fb8b58f384ebdd3186#diff-58df978638a78c3d9ed64d51ba83a775R22

Reply:
I agree to @shyim cause this solution is slowing down compiling theme 🤔

Reply:
Mh, but when creating a child class of the autoprefixer, one either needs to do it twice as well, or implement everything what is in the base class?

I think this is more aimed that one does not forget it later on, right? And has an easier way to remove it later on.